client/src/RealatedComparisonComponents/ComparisonModal.jsx

```diff
@@ -27,7 +27,11 @@ class ComparisonModal extends React.Component {
   remapFeaturesInfo (rawProductFeatures) {
     var mappedFeatures = {};
     rawProductFeatures.forEach((featureObj) => {
-      mappedFeatures[featureObj.feature] = featureObj.value;
+      if (featureObj.value === 'null') {
```

Comment:
Does this cover _null_ and 'null'? I found a case today in my module where they were throwing me null and 'null' and it broke my component lol

Reply:
no harm in doing both I suppose.